dirty_cat/_minhash_encoder.py

```diff
@@ -89,9 +90,7 @@ def _more_tags(self) -> Dict[str, List[str]]:
         """
         return {"X_types": ["categorical"]}
 
-    def minhash(
-        self, string: str, n_components: int, ngram_range: Tuple[int, int]
-    ) -> np.array:
+    def get_murmur_hash(self, string: str) -> np.array:
```

Comment:
The method is still public !
```suggestion
    def _get_murmur_hash(self, string: str) -> np.array:
```
(references need to be renamed as well)

Reply:
Yes, I think it should stay this way. Take a look at my comment above. This method is equivalent for `get_fast_hash` but for the `murmur` method. Seems to have no problem being public.

Reply:
If we make this one private, then the `get_fast_hash` should also be made private.

Reply:
Ok I saw the comment below.